tests/contrib/psycopg/test_psycopg.py

```diff
@@ -2,7 +2,7 @@
 import time
 from unittest import skipIf
 
-# 3p
```

Comment:
This comment was removed because it is useless and messes with isort linting. Here's where it was introduced: https://github.com/DataDog/dd-trace-py/pull/39/files#diff-41f07c055df91a25d2f3c15e89367264e67c74167119355a7b1f8f9da3bf9546